mne/preprocessing/ica.py

```diff
@@ -1219,8 +1226,15 @@ def find_bads_ref(self, inst, ch_name=None, threshold=3.0, start=None,
             Low pass frequency.
         h_freq : float
             High pass frequency.
+        method : {'together', 'separate'}
+            Method to use to identify reference channel related components.
+            Defaults to "together." See notes.
```

Comment:
```suggestion
            Defaults to "together." See notes.

            .. versionadded:: 0.20
```

Reply:
... and might as well have this param added after `reject_by_annotation` to keep arg order